utils/gear-logger/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think we either need types or don't need typescript at all

Reply:
Yes, I mean that if don't build types than we don't need typescript here at all. This package can be just a js package. So is the `logger` variable exported from this package typed?